vortexor/Readme.md

```diff
@@ -145,3 +145,92 @@ Operators can decide whether to adopt Vortexors without concerns about network
 protocol changes. Upgrading involves specifying the Vortexor's TPU address and
 verified packet receiver network address via CLI or Admin RPC. The transition is
 designed to be seamless for operators.
+
+# Pair a Vortexor and Validator
+
+To pair a validator and Vortexor, follow these steps:
+
+---
+
+### Step 1: Run the Vortexor
+Run the Vortexor using the following command:
+
+```bash
+solana-vortexor --identity /path/to/id.json \
+    --destination <validator_receiver_address> \
+    --dynamic-port-range <port_range> \
+    --rpc-server <rpc_server_address> \
+    --websocket-server <websocket_server_address>
+```
+
+**Parameters:**
+- `--identity`: Path to the identity keypair file for the Vortexor.
+- `--destination`: The validator's receiver address where verified packets will be sent (e.g., `10.138.0.136:8100`).
+- `--dynamic-port-range`: The port range used by the Vortexor for TPU traffic (e.g., `9200-9300`).
+- `--rpc-server`: The RPC server address to fetch cluster information (e.g., `http://10.138.0.137:8899`).
+- `--websocket-server`: The WebSocket server address to fetch stake information (e.g., `ws://10.138.0.137:8900`).
+
+**Example:**
+
+In the example below, we are pairing the vortexor with the validator running on 10.138.0.136 and we use
+the RPC node 10.138.0.137 for RPC and websocket service:
+
+```bash
+solana-vortexor --identity /home/solana/.config/solana/id.json \
+    --destination 10.138.0.136:8100 \
+    --dynamic-port-range 9200-9300 \
+    --rpc-server http://10.138.0.137:8899 \
+    --websocket-server ws://10.138.0.137:8900
+```
+
+---
+
+### Step 2: Find the Vortexor's TPU and Forward Addresses
+The Vortexor's TPU and forward addresses can be found in its log file. For example:
+
+```
+[2025-04-24T17:40:13.098760226Z INFO  solana_vortexor] Creating the Vortexor. The tpu socket is: Ok(0.0.0.0:9200), tpu_fwd: Ok(0.0.0.0:9201)
```

Comment:
This is a lot of messy math that people will 100% get wrong. Not the problem of this readme as such, just this UX is too hard to get right.

Reply:
Will update the vortexor implementation in future PRs to make it more clear.

Reply:
https://github.com/anza-xyz/agave/issues/6046